frappe/public/js/frappe/views/communication.js

```diff
@@ -640,10 +640,14 @@ frappe.views.CommunicationComposer = Class.extend({
 				this.message = localStorage.getItem(doctype + docname) || '';
 			}
 		}
-
-		if(this.real_name) {
-			this.message = '<p>'+__('Dear') +' '
-				+ this.real_name + ",</p><!-- salutation-ends --><br>" + (this.message || "");
+		
+		const SALUTATION_END_COMMENT = "<!-- salutation-ends -->";
+		if (this.real_name && !message.includes(SALUTATION_END_COMMENT)) {
+			this.message = `
+				<p>${__('Dear')} ${this.real_name},</p>
```

Comment:
Thanks for fixing! By the way, `__('Dear')` is impossible to translate correctly without knowing how it is used here. `__('Dear {},', [this.real_name], 'Salutation in Email')` would be a bit easier.

Reply:
Fixed with #13503